python/CLOUDFRONT_VIEWER_POLICY_HTTPS/CLOUDFRONT_VIEWER_POLICY_HTTPS.py

```diff
@@ -0,0 +1,366 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+"""
+Description:
+  Checks whether your Amazon CloudFront Distributions use HTTPS (directly or via a redirection). The rule returns NON_COMPLIANT if the ViewerProtocolPolicy is set to 'allow-all' (i.e Viewers can use HTTP or HTTPS).

```

Comment:
Can you add `in the defaultCacheBehavior or in the cacheBehaviors`.

`Checks whether your Amazon CloudFront Distributions use HTTPS (directly or via a redirection). The rule returns NON_COMPLIANT if the ViewerProtocolPolicy is set to 'allow-all' in the defaultCacheBehavior or in the cacheBehaviors (i.e Viewers can use HTTP or HTTPS).`